--- ../cache/rustc-1.22.1-src/src/bootstrap/configure.py	2017-11-22 21:33:00.000000000 +0000
+++ ./src/bootstrap/configure.py	2017-11-24 06:58:51.213161882 +0000
@@ -217,11 +217,6 @@
         unknown_args.append(arg)
 p("")
 
-if 'option-checking' not in known_args or known_args['option-checking'][1]:
-    if len(unknown_args) > 0:
-        err("Option '" + unknown_args[0] + "' is not recognized")
-    if len(need_value_args) > 0:
-        err("Option '{0}' needs a value ({0}=val)".format(need_value_args[0]))
 
 # Parse all known arguments into a configuration structure that reflects the
 # TOML we're going to write out
